test/utils/math/Math.t.sol

```diff
@@ -206,23 +206,43 @@ contract MathTest is Test {
         } catch {}
     }
 
+    // MOD EXP
+    function testModExp(uint256 b, uint256 e, uint256 m) public {
+        try this.modexp(b, e, m) returns (uint256 result) {
```

Comment:
Why does this have to be a external call (`this.modex`) and not just `Math.modExp`?
The revert can be expected conditionally if `m == 0` with:

```
if(m == 0) vm.expectRevert(...)
```

Reply:
Can you expect revert internal library operation? I was under the impression that you can only expectRevert actual calls/static calls ... which is why we have this wrapper.

Note we do the same in mulDiv